packages/hash/api/src/collab/Instance.ts

```diff
@@ -886,6 +792,7 @@ export const getInstance =
     if (forceNewInstance || instances[pageEntityId]?.errored) {
       instances[pageEntityId]?.stop();
       delete instances[pageEntityId];
+      instanceCount--;
```

Comment:
This fixes an unrelated bug where the instance count gets out of sync sometimes. This was an oversight.